packages/compartment-mapper/src/types/typescript.d.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file Helpers relevant to any TypeScript project.
```

Comment:
there might be a copyright/license notice we need to include from [type-fest](https://npm.im/type-fest), as these types were copied from it.

Reply:
Thank you for calling this out. `type-fest` is CC0, so no attribution is required. I’m inclined to add a citation regardless, but doesn’t need a copy of the license.